processor/error/payload_test.go

```diff
@@ -80,7 +78,7 @@ func TestPayloadTransform(t *testing.T) {
 		outputEvents := test.Payload.transform()
 		for j, outputEvent := range outputEvents {
 			assert.Equal(t, test.Output[j], outputEvent.Fields, fmt.Sprintf("Failed at idx %v; %s", idx, test.Msg))
-			assert.Equal(t, expectedTime, outputEvent.Timestamp)
+			assert.WithinDuration(t, time.Now(), outputEvent.Timestamp, time.Second)
```

Comment:
I suggest to rather store `Time.now()` as a variable somewhere and check the exact time. This looks like it could become a flaky test very easily. 

Do you know the `timecop` tools? I know it from ruby, but saw that there is also a [go-timecop](https://github.com/bluele/go-timecop) implementation. This can be quite useful when testing something time related (probably not necessary here but worth to mention.)

Reply:
done!